spec/factories/tool_draft_factories.rb

```diff
@@ -8,4 +8,208 @@
     short_name { nil }
     entry_title { nil }
   end
+
+  # factory :invalid_tool_draft, class: ToolDraft do
```

Comment:
I think this should either have a comment explaining what needs to be added or be removed